crates/recursor/src/error.rs

```diff
@@ -166,13 +182,26 @@ impl From<Error> for String {
 
 impl From<ResolveError> for Error {
     fn from(e: ResolveError) -> Self {
-        if let Some(ProtoErrorKind::NoRecordsFound { soa, ns, .. }) =
-            e.proto().map(ProtoError::kind)
+        if let Some(ProtoErrorKind::NoRecordsFound {
+            query,
+            soa,
+            ns,
+            authorities,
+            ..
+        }) = e.proto().map(ProtoError::kind)
```

Comment:
As discussed before/elsewhere, we have ownership of the `ResolveError`, so there's not a good reason to actually clone here.